### DISCUSSION TITLE
My github page is available only for me, it's not available for others(it shows 404 error to others)

### DISCUSSION THREAD

**Discussion by Rajeshkannan-Muthukumar** [community/community#59871] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My github page is available only for me,when i logged in my account . it's not available for others(it shows 404 error to others)
![image](https://github.com/community/community/assets/93901857/74cd473d-45c1-4199-a026-c11defc592bd)

**Comment by cehfisher** [comment]
Hi @Rajeshkannan-Muthukumar - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category and close it out as a duplicate of #59935? Thanks!
